doc/expressionLanguage.md

```diff
@@ -175,3 +176,35 @@ The following table shows expressions and their expected outcomes for a measure
 | '@value * 5.2'              | 31.2                  |
 | '"Pruebas " + "De Strings"' | 'Pruebas De Strings'  |
 | '@name value is @value'     | 'DevId629 value is 6' |
+
+## <a name="ngsiv2"/> NGSIv2 support
+
+As it is explained in previous sections, expressions can have two return types: String or Number, being the former one the default. Whenever a expression is executed without error, its result will be cast to the configured type. 
+
+On one hand, in NGSIv1 since all attributes'values are string, in the expression parser the expression type is set always to String and the transformation of the information coming from the SouthBound is done using replace instruction. Therefore, values sent to the CB will always be Strings. This can be seen in (#execution) example.
```

Comment:
Typo: attributes'values

Reply:
Fixed in 776fe93a154bdd014f5bb7b37c8e165a52ab66d1